packages/core/src/internal/clients/pubsub/AbstractPubsubClient.ts

```diff
@@ -28,6 +29,7 @@ export interface SendSubscribeOptions {
     error: TopicSubscribe.Error,
     subscription: TopicSubscribe.Subscription
   ) => void;
+  onDiscontinuity?: (discontinuity: TopicDiscontinuity) => void;
```

Comment:
Since this is internal I think you could probably make it required like the other ones?